dynamic-config/config-apollo/src/main/resources/META-INF/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
no need to provide LICENSE.txt and NOTICE.txt, maven plugin will generate it automatically